src/pages/steps/PriceInformation.tsx

```diff
@@ -154,7 +154,7 @@ const PriceInformation = ({
     watch,
     setValue,
     trigger,
-    formState: { errors },
+    formState: { errors, touchedFields },
```

Comment:
I'm still not sure about this. But does it work with `dirtyFields` since we haven't had to use touchedFields yet

Reply:
Jup it also works with `dirtyFIelds` 👍